### PR TITLE
README.md: Use documentation from kernelci.org/docs 

### PR DESCRIPTION
**Pull request by JenySadadia**
Add links to https://kernelci.org/docs for the
setup and usage instructions and remove duplicate
information from the file.

Signed-off-by: Jeny Sadadia <jeny.sadadia@collabora.com>

### CODE REVIEW

**Review by gctucker — COMMENTED** [review]
Erm, no the idea was to remove everything from the README that is now already in the docs.

**Comment by mgalka** [comment]
> > Erm, no the idea was to remove everything from the README that is now already in the docs.
> 
> Oh, in my understanding I needed to add missing things.

Oh, it looks it's my fault. That's how I understood the intention, too. If the idea was to get rid of the parts covered int he docs, maybe we could add a link to the documentation in the README files?

**Comment by JenySadadia** [comment]
> 

Okay. Yes, I will send the changes.

File: README.md [hunk 1]
```diff
@@ -2,70 +2,23 @@ KernelCI Pipeline
 -----------------
 
 Modular pipeline based on the new [KernelCI
-API](https://github.com/kernelci/kernelci-api).
+API](https://github.com/kernelci/kernelci-api).
```
Comment (gctucker) [inline README.md:5]: Superfluous space at the end of the line

File: README.md [hunk 2]
```diff
@@ -2,70 +2,23 @@ KernelCI Pipeline
 -----------------
 
 Modular pipeline based on the new [KernelCI
-API](https://github.com/kernelci/kernelci-api).
+API](https://github.com/kernelci/kernelci-api). 
 
-To use it, first start the API.  Then start the services in this repository on
-the same host:
+Please refer to the pipeline design here: https://kernelci.org/docs/api/overview/#pipeline-design
```
Comment (gctucker) [inline README.md:7]: Like with the API README changes, it's better to use links:
```suggestion
Please refer to the [pipeline design documentation](https://kernelci.org/docs/api/overview/#pipeline-design) for more details.
```

File: README.md [hunk 3]
```diff
@@ -2,70 +2,23 @@ KernelCI Pipeline
...
-notifier    | 2021-12-08 23:01:38.717247  2a987e65025e  Pass      check-describe
-^CGracefully stopping... (press Ctrl+C again to force)
-Stopping runner   ... done
-Stopping notifier ... done
-```
+To use it, first, start the API.  Then start the services in this repository on the same host.
 
-The `trigger` service was run only once as it's not currently configured to run
-periodically.
+Follow instructions from https://kernelci.org/docs/api/getting-started/#setting-up-a-pipeline-instance to add a token and then to start the services.
```
Comment (gctucker) [inline README.md:11]: Ditto

**Review by gctucker — APPROVED** [review]

**Pull request merged**